src/env/env.h

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*   tosh - 21 Shell                                      ::::::::            */
+/*                                                      :+:    :+:            */
+/*   By: aholster <aholster@student.codam.nl>          +:+                    */
+/*       ivan-tey <ivan-tey@student.codam.nl>         +#+                     */
+/*       nloomans <nloomans@student.codam.nl>        +#+                      */
+/*                                                 #+#    #+#                 */
+/*   License: GPLv3                                ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef ENV_H
+# define ENV_H
+
+# include <libft.h>
+# include <stdlib.h>
+
+struct					s_env_list
+{
+	char				*key;
+	char				*value;
+	t_list_conn			conn;
+};
```

Comment:
I find the name `s_env_list` confusing, as this struct refers to a single element instead of the entire list. How about `s_env_pair`?